var/spack/repos/builtin/packages/jq/package.py

```diff
@@ -34,3 +34,4 @@ class Jq(AutotoolsPackage):
     version('1.5', 'c8070bd6ec275404f77db3d2e568c9a3')
 
     depends_on('oniguruma')
+    depends_on('bison@3.0:')
```

Comment:
If you're on Linux, I would run `ldd -r` on all of the executables/shared libraries and see if any of them link to `bison`. If not, it's a build dependency. Conversely, if you are able to reproduce the problem where it crashes when it can't find bison, see if adding it as a build dependency only works. It's easy to do in a VM where you can install/uninstall bison on the system.